### ISSUE TITLE
Images not showing up

### ISSUE THREAD

**Issue by ag3ntofkarma** [Pariah-Zero/dead-in-thay#1]
When I install the module, the icons and everything flows in fine, but there are no map images.

**Comment by Pariah-Zero** [comment]
I imagine you're using FVTT 11?

I'll spin out a new world and see if I can replicate your issue.

**Comment by ag3ntofkarma** [comment]
Yea, 11 stable. What is weird is that it is not throwing any errors at all, just not bringing them in.

**Comment by Pariah-Zero** [comment]
I believe I've found the problem. To verify:  Pick a scene (I don't think it matters which one, but 'Abyssal Prisons' is usually the starting point.

Copy/Paste the value in the `Background Image` field - I'm betting it's `modules/dead-in-thay/assets/Fiendish-Arena.webp`

I'm also going to verify that's the case versus what's here in GitHub (so far, it looks to be the case) and a 'partial' update to file paths needs to be made 'full'. (ie. the paths are not correct). (insert facepalm) I'll get them updated and work on getting the update pushed out.

I'll let you know when I'm ready so you can test the update before I make the release official.

**Comment by ag3ntofkarma** [comment]
Awesome, thank you very much and 100% for the quick response. We are on Vacation and decided to throw something together and this was the chosen dungeon. I appreciate it and I can test run the update whenever you get it rolling (don't rush on my behalf).

**Comment by Pariah-Zero** [comment]
> Awesome, thank you very much and 100% for the quick response. We are on Vacation and decided to throw something together and this was the chosen dungeon. I appreciate it and I can test run the update whenever you get it rolling (don't rush on my behalf).

Well, if YOU are in a hurry, it appears that what you can do is relatively simple (if tedious)

Change the path for the `Background Image` from `modules/dead-in-thay/assets/` → `modules/p0-dead-in-thay/assets/`

(Leave the rest of the filename alone - which really is just `<scenename.webp>`).

Let me know if that works for you - I'm assuming that's the issue on your end (it's what I see) - and that's what I'm fixing.

**Comment by ag3ntofkarma** [comment]
> > Awesome, thank you very much and 100% for the quick response. We are on Vacation and decided to throw something together and this was the chosen dungeon. I appreciate it and I can test run the update whenever you get it rolling (don't rush on my behalf).
> 
> Well, if YOU are in a hurry, it appears that what you can do is relatively simple (if tedious)
> 
> Change the path for the `Background Image` from `modules/dead-in-thay/assets/` → `modules/p0-dead-in-thay/assets/`
> 
> (Leave the rest of the filename alone - which really is just `<scenename.webp>`).
> 
> Let me know if that works for you - I'm assuming that's the issue on your end (it's what I see) - and that's what I'm fixing.

Yes that did work well. I will spin up another world when you get to throwing in a fix to check it for you. Thank you again.
